packages/react-magma-dom/src/components/Select/shared.ts

```diff
@@ -1,11 +1,17 @@
-import { inputBaseStyles } from '../InputBase';
-import { Card } from '../Card';
-import { transparentize } from 'polished';
+import { css } from '@emotion/react';
 import styled from '@emotion/styled';
+import { transparentize } from 'polished';
 import { ThemeInterface } from '../../theme/magma';
-import { css } from '@emotion/react';
+import { Card } from '../Card';
+import { inputBaseStyles } from '../InputBase';
 
 function buildListHoverColor(props) {
+  if (props.isDisabled) {
+    if (props.isInverse) {
+      return transparentize(0.6, props.theme.colors.neutral100);
+    }
+    return transparentize(0.4, props.theme.colors.neutral500);
+  }
```

Comment:
I don't think we want to change the background/hover color. Instead, we should just change the text color similar to how we do it in Dropdown:
![image](https://github.com/user-attachments/assets/aa687e73-ae6d-46b2-b963-a30208206fbd)
![image](https://github.com/user-attachments/assets/9332acb7-c255-4210-a30e-811af1407555)